update keep on latest version beta (#5025)
dbt-labs/docs.getdbt.com: website/docs/docs/dbt-versions/upgrade-dbt-version-in-cloud.md

You can upgrade to **Keep on latest version** no matter which version of dbt you currently have selected. As a best practice, dbt Labs recommends that you test the upgrade in development first; use the [Override dbt version](#override-dbt-version) setting to test _your_ project on the latest dbt version before upgrading your deployment environments and the default development environment for all your colleagues.

:::tip Interested? 

> This feature is currently available in beta for all dbt-Labs-maintained adapters; rolling out to wider availability in March.

Reviewer: ```suggestion
This feature is currently available in beta for all dbt Labs-maintained adapters and will be widely available in March.
```
